### DISCUSSION TITLE
Welcome to goxel Discussions!

### DISCUSSION THREAD

**Discussion by guillaumechereau** [guillaumechereau/goxel#310] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YarlBoro** [comment]
Hello! I'm Borion, voxel enthusiast & long time Goxel user. I hope to meet other voxel fans and discuss with them workflows, inspirations & future of Goxel :)

**Comment by pegvin** [comment]
i hope this software gets more attention, Love Live OSS & Contributors.

**Comment by Dolampo4ki** [comment]
Hello!
Tell me, will the program be updated on android (last version 0.10.4)?
p.s. Very inconvenient menu of the left sidebar (if you use many layers, then you need to scroll the menu itself with editor icons, you need to rewind a lot to continue editing the object (eye icon in the three layers menu) :(
p.s.s. on a 6.2-inch smartphone (on the top left, next to the 6 squares icon, there is no way to move further and click on About (scroll the line itself to select the icon you need next in the editor

**Comment by guillaumechereau** [comment]
I haven't done any update of the mobile versions for a while.  I'll try to make one this week!

For the UI, I agree that currently this is not optimal.  I am thinking of redoing the UI from scratch to make more mobile friendly at some point.

**Comment by willrcoding** [comment]
I've finally decided on giving this mobile version a try. Very happy to see what I can do after toying with this for a few minutes.

**Comment by guillaumechereau** [comment]
Thanks!  I am working on an update to the mobile version to improve its
UI.  Hopefully I can release it sometime next month if i get the time.

**Comment by NimeNayota** [comment]
Hello. I recently started using Goxel and really liked its usability. I wonder if it supports the pressure sensitivity of a graphics tablet? Pressure sensitivity makes creating dynamic shapes faster. 
![image](https://github.com/user-attachments/assets/f1d37457-4af0-44ae-b4d1-2e1e00820610)

**Comment by guillaumechereau** [comment]
Not at the moment unfortunately.  You would like the pressure to dynamically change the size of the brush?  I could try to add that in a future update.

**Reply by NimeNayota** [reply]
Yes, that would be a very useful feature. And also color transparency if possible.
